modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -58,3 +58,19 @@ The following lists specify the default permissions for control plane and comput
 * `ec2:DescribeInstances`
 * `ec2:DescribeRegions`
 ====
+
+.Optional permissions for using custom KMS key during installation
+[%collapsible]
+====
+* `kms:Decrypt`
+* `kms:Encrypt`
+* `kms:GenerateDataKey`
+* `kms:GenerateDataKeyWithoutPlainText`
+* `kms:DescribeKey`
+* `kms:RevokeGrant`
+* `kms:CreateGrant`
+* `kms:ListGrants`
+* `kms:ListGrants`
+* `kms:ListGrants`
+* `kms:ListGrants`
```

Comment:
@Chin2691 There are duplicate `kms:ListGrants` permissions in the list, please double check the list in [permissions.go](https://github.com/openshift/installer/blob/882a2943d87442cb4955a127e4cdf3940216d785/pkg/asset/installconfig/aws/permissions.go#L253-L262), thanks